examples/pub-sub-1-topic/subscriber.js

```diff
@@ -1 +0,0 @@
-'use strict'
```

Comment:
As the example was not even complete and we already have a pubsub example on `js-libp2p`, I think that it is not worth to have another one to maintain here. 

Ideally, we should look at https://github.com/libp2p/js-libp2p-examples as the way to go? Once I get to automation of the interop for `js-libp2p`, I will also look into adding CI jobs for examples and we should probably migrate all examples to there and like in the appropriate repos the examples